### DISCUSSION TITLE
[Open] Add treasury to the "Resource list"

### DISCUSSION THREAD

**Discussion by MrNemo64** [ArcanePlugins/Treasury#141] [Ideas]
A while ago I created [this](https://www.spigotmc.org/wiki/resources-list/) spigot wiki page. It is ment to be an index/list of all the apis and frameworks plugin developers have created, to make it easyer to check if something already existed without having to spend hours searching it.
I think we should add it to the page but since it's scope is going to keep on growing I really don't know how to post it

**Comment by lokka30** [comment]
> A while ago I created [this](https://www.spigotmc.org/wiki/resources-list/) spigot wiki page. It is ment to be an index/list of all the apis and frameworks plugin developers have created, to make it easyer to check if something already existed without having to spend hours searching it. I think we should add it to the page but since it's scope is going to keep on growing I really don't know how to post it

I don't think Treasury fits any of the categories on there. Should it be listed multiple times, once under each category it applies to?

**Reply by MrNemo64** [reply]
Since right now we only have the economy API we could add it to the Utilities
